TryGhost/Ghost
# 🎨 Added Koenig Demo Post

no issue

- add new migration for 1.25 to insert the draft demo post
- tested on sqlite3 + mysql
- add the new demo post to our fixtures to ensure a new blog get's the demo post as well
- ensure demo post is never inserted twice

---
- [x] add handling if ghost author user doesn't exist anymore
